virtio-devices/src/balloon.rs

```diff
@@ -94,6 +95,42 @@ pub struct VirtioBalloonConfig {
     actual: u32,
 }
 
+#[derive(Copy, Clone, Debug)]
+struct PartPage {
```

Comment:
Given this will only be used for balloon "inflation", I think reuse the name from qemu's patch is better. Please rename it to `PartiallyBalloonedpage`

Reply:
OK